### PR TITLE
Add CVEs for various new Metasploit modules

### PR DESCRIPTION
**Pull request by todb-r7**
This covers some CVEs from some recent Metasploit modules. Can I get a spot check please? Mostly looking for failures in copy-pasting. :)

### CODE REVIEW

File: 2020/7xxx/CVE-2020-7352.json [hunk 1]
```diff
@@ -1,18 +1,110 @@
...
+    ],
+    "data_format": "MITRE",
+    "data_type": "CVE",
+    "data_version": "4.0",
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "The GalaxyClientService component of GOG Galaxy runs with elevated SYSTEM privileges in a Windows environment. Due to the software shipping with embedded, static RSA private ley, an attacker with this key material and local user permissions can effectively send any operating system command to the service for execution in this elevated context. The service listens for such commands on a locally-bound network port, localhost:9978. A Metasploit module has been published with exploits this vulnerability. This issue affects the 2.0.x branch of the software (2.0.12 and earlier) as well as the 1.2.x branch (1.2.64 and earlier). A fix was issued for the 2.0.x branch of the affected software."
```
Comment (djordan-r7) [inline 2020/7xxx/CVE-2020-7352.json]: "embedded, static RSA private ley", should be "key", right?

Comment (djordan-r7) [reply 2020/7xxx/CVE-2020-7352.json]: Also, "A Metasploit module has been published with exploits this vulnerability", should be "which exploits this vulnerability"?

**Review by djordan-r7 — COMMENTED** [review]
Hey @todb-r7 - left a few comments with some minor suggestions on :)

File: 2020/7xxx/CVE-2020-7352.json (hunk 1 above)
Comment (todb-r7) [reply 2020/7xxx/CVE-2020-7352.json]: Derp yep on both.

**Pull request merged**